draft-barnes-hpke-hpke.md

```diff
@@ -1783,10 +1861,17 @@ Template:
 * Value: The two-byte identifier for the algorithm
 * KDF: The name of the algorithm
 * Nh: The output size of the Extract function in bytes
+* Two-Stage: Whether this is a two-stage KDF.  Allowed values are:
+    * "Y" - This is a two-stage KDF
+    * "N" - This is a single-stage KDF
```

Comment:
Does this need to be in the registry, or is it an implementation detail?

Reply:
Fair point, I think we can get away without it.